All_Relationships/.ipynb_checkpoints/1.data-loader-checkpoint.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
This file is in an `.ipynb_checkpoints` directory. This is likely a temp file created by Jupyter and should be ignored?

Let's add the following to .gitignore

```
# Jupyter Notebook
.ipynb_checkpoints
```